plugins/coverimage.koplugin/main.lua

```diff
@@ -0,0 +1,153 @@
+local Device = require("device")
+
+if Device.isKindle == nil and Device.isKobo == nil then
```

Comment:
you can also disable it on cervantes, remarkable, desktop ...

Or just enable it on android (and the emulator for testing)

Reply:
I have disabled the plugin on all devices except Android and Emulator